mediaplayerbackend.cpp

```diff
@@ -270,21 +277,23 @@ void MediaPlayerBackend::onCurrentTlTrackReceived(const mopidy::TlTrack &tlTrack
 
 void MediaPlayerBackend::onStateReceived(mopidy::PlaybackState state)
 {
-    qDebug() << "onStateReceived";
     switch (state) {
     case mopidy::PlaybackState::Paused:
+        qDebug() << "onStateReceived: Paused";
         m_playbackState = QIviMediaPlayer::Paused;
         m_playbackController.getCurrentTlTrack();
         m_positionTrackerTimer.stop();
         break;
 
     case mopidy::PlaybackState::Playing:
+        qDebug() << "onStateReceived: Playing";
         m_playbackState = QIviMediaPlayer::Playing;
         m_playbackController.getCurrentTlTrack();
         m_playbackController.getTimePosition();
         break;
 
     case mopidy::PlaybackState::Stopped:
+        qDebug() << "onStateReceived: Stopped";
```

Comment:
pl add default case, and print qDebug() + the value

Reply:
what would default case do? value of what?

Reply:
For example:
default:
  print qWarning() << "onStateReceived: error, state not supported: " << state;
  break;